components/resources/wallet_strings.grdp

```diff
@@ -329,8 +329,8 @@
   <message name="IDS_BRAVE_WALLET_CONNECT_HARDWARE_PANEL_DISCONNECTED" desc="Connect Hardware panel disconnected status"><ph name="DEVICE_NAME"><ex>Ledger</ex>$1</ph> disconnected</message>
   <message name="IDS_BRAVE_WALLET_CONNECT_HARDWARE_PANEL_INSTRUCTIONS" desc="Connect Hardware panel instructions button">Instructions</message>
   <message name="IDS_BRAVE_WALLET_CONNECT_HARDWARE_PANEL_CONNECT" desc="Connect Hardware panel title">Connect your <ph name="DEVICE_NAME"><ex>Ledger</ex>$1</ph></message>
-  <message name="IDS_BRAVE_WALLET_CONNECT_HARDWARE_PANEL_CONFIRMATION" desc="Connect Hardware panel confirmation description">Hardware wallet requires confirmation on device. <ph name="DEVICE_NAME"><ex>Ledger</ex>$1</ph></message>
```

Comment:
This change is unrelated to most of the pr: I removed the "." here because I think it was accidentally put there.

Changes from

> Hardware wallet requires confirmation on device Ledger

To

>  Hardware wallet requires confirmation on device Ledger